target/linux/rtl838x/files-5.4/drivers/net/dsa/rtl838x_sw.c

```diff
@@ -30,34 +31,46 @@ struct rtl838x_mib_desc {
 	const char *name;
 };
 
-inline void rtl838x_mask_port_reg(u64 clear, u64 set, int reg)
+static inline void rtl838x_mask_port_reg(u64 clear, u64 set, int reg)
 {
 	sw_w32_mask((u32)clear, (u32)set, reg);
```

Comment:
Because `sw_*` are implemented as macros, it's not obvious, but these u32-casts don't have any effect.

I suggest reimplementing `sw_*` as inline functions in a follow-up PR to make clear which types are expected.